tests/diagnostics/ctor-noncopyable-retval-legal.slang

```diff
@@ -0,0 +1,103 @@
+//TEST:SIMPLE(filecheck=CHECK):
```

Comment:
Is this still necessary? seems superseded by the runnable tests.

Reply:
i left them behind because it explicitly listed all the test cases, including the infinite loop case. shall i remove them?

Reply:
I see, it is fine to leave them here.